.resources/strings.json

```diff
@@ -195,46 +218,52 @@
 		"ja": "表示",
 		"es": "Vista",
 		"de": "Anzeigen"
+		"pt-BR": "Ver"
```

Comment:
Not sure if this is some kind of special `:` character, it's highlighted in the pull request on Github. If so can you change it to a normal colon character before I merge this in?